### PR TITLE
sidebar styling changes

### PR DESCRIPTION
**Pull request by Nnaga1**
**Summary:** Summary of changes

Addresses [CUMULUS-2142: Update Sidebar Styling](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2142)

## Changes
-Changed styling of Sidebar button while also adding Tooltips instead of the basic title tags

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

File: app/src/css/_buttons.scss [hunk 1]
```diff
@@ -69,11 +69,12 @@ $delete-btn-hover-bg-color: darken($error-red, 10%);
   }
 
   &--round {
+    stroke: 1px solid #ffffff;
     border-radius: 50%;
     height: 2em;
     width: 2em;
-    background-color: $midnight-blue;
-    border: none;
+    background-color:#1E6B9D;
```
Comment (Alissonsleal) [inline app/src/css/_buttons.scss:76]:
```suggestion
    background-color: $sub-blue;
```

**Review by Alissonsleal — APPROVED** [review]

File: app/src/css/_buttons.scss [hunk 2]
```diff
@@ -69,11 +69,12 @@ $delete-btn-hover-bg-color: darken($error-red, 10%);
   }
 
   &--round {
+    stroke: 1px solid #ffffff;
```
Comment (jjmccoy) [inline app/src/css/_buttons.scss:72]: @Nnaga1 You can remove stroke as well.

**Review by jjmccoy — CHANGES_REQUESTED** [review]
Looks great! Nice CSS work and incorporating the tooltip.

**Review by jjmccoy — APPROVED** [review]

**Comment by jjmccoy** [comment]
LGTG! Thanks, @Nnaga1 for the quick changes.

**Pull request merged**